__tests__/app.test.js

```diff
@@ -70,3 +71,63 @@ describe.only("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("indicates how much the votes property in the database should be updated by", () => {
+    return request(app)
+      .patch(`/api/articles/1`)
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 101,
+        });
+      });
+  });
+  test("status: 400, responds with an error message when a bad request is passed", () => {
+    return request(app)
+      .patch(`/api/articles/!£#*`)
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("This is an invalid url");
+      });
+  });
+  test("status: 404, responds with an error message when trying to patch an article ID that does not exist", () => {
+    return request(app)
+      .patch(`/api/articles/1111`)
+      .send({ inc_votes: 1 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Article not found");
+      });
+  });
+});
+
```

Comment:
think about how you'd test to account for the object that's sent to patch too:
- what if the inc_votes value is invalid?
- what if the object has no inc_votes key at all?